build-ventus.sh

```diff
@@ -204,12 +211,29 @@ build_icd_loader() {
   make && make install
 }
 
+# Test the rodinia test suit
+test_rodinia() {
+   cd ${RODINIA_DIR}
+   make OCL_clean
+   make OPENCL
+}
+
+# Test some test cases of the pocl
+test_pocl() {
+   cd ${POCL_BUILD_DIR}/examples
+   ./vecadd/vecadd
+   ./matadd/matadd
```

Comment:
Add TODO here , because later we need to append more test cases

Reply:
done